align_anything/configs/template.py

```diff
@@ -705,15 +653,15 @@ def format_sample(self, raw_sample: dict[str, Any], path: str=None) -> dict[str,
         }
 
     
-@register_template('Llava')
-class Llava:
+@register_template('LLAVA')
+class LLAVA:
```

Comment:
Modify the corresponding template in the document, for example this [sample](https://align-anything.readthedocs.io/en/latest/training/text_image_to_text.html)